docs/getting-started/creating-plugins.md

```diff
@@ -35,6 +41,53 @@ The `json` file contains the following structure:
 }
 ```
 
+#### Definition structure
+
+The `Definition` property has the following structure:
+
+```json
+{
+  "Id": "Id shall contain a stable identifier for the rule",
+  "Name": "Name may contain an identifier that is understandable to an end user",
+  "HelpUri": "HelUri may contain the absolute URI of the primary documentation for the reporting item",
```

Comment:
>HelUri [](http://example.com/codeflow?start=14&length=6)

this should be 'HelpUri'.

In general, you should encapsulate variable and api names.

'HelpUri' may contain the absolute URI of the primary documentation for the reporting item.